src/visualization/plot_recipes.jl

```diff
@@ -696,6 +706,88 @@ end
   end
 end
 
+const DGMultiSemidiscretizationHyperbolic{Mesh, Equations} =
+  SemidiscretizationHyperbolic{Mesh, Equations, InitialCondition, BoundaryCondition, SourceTerms,
+  <:DGMulti, Cache} where {InitialCondition, BoundaryCondition, SourceTerms, Cache}
+
+# overloads PlotData2D but returns a DGMultiPlotData for plotting DGMulti solutions
+function PlotData2D(u::StructArray, semi::DGMultiSemidiscretizationHyperbolic;
+                    solution_variables=nothing, grid_lines=true)
+  rd = semi.solver.basis
+  md = semi.mesh.md
+  equations = semi.equations
+  solution_variables_ = digest_solution_variables(equations, solution_variables)
+  variable_names = SVector(varnames(solution_variables_, equations))
+  return DGMultiPlotData(u, equations, solution_variables_, variable_names, rd, md)
+end
+
+# need to define this function because some keywords from the more general plot recipe
+# are not supported (e.g., `max_supported_level`).
+@recipe function f(u::StructArray, semi::DGMultiSemidiscretizationHyperbolic;
+                   solution_variables=cons2cons, grid_lines=true)
+  return PlotData2D(u, semi)
+end
+
+@recipe function f(pds::PlotDataSeries2D{<:DGMultiPlotData{2}})
+
+  pd = pds.plot_data
+  @unpack u, equations, solution_variables, rd, md = pd
+  @unpack variable_id = pds
+
+  @unpack Vp = rd
+  mul_by_Vp!(out,x) = mul!(out, Vp, x)
+
+  num_plotting_points = size(Vp, 1)
+  nvars = nvariables(equations)
+  u_plot_local = StructArray{SVector{nvars, eltype(first(u))}}(ntuple(_->zeros(num_plotting_points), nvars))
+  u_plot = zeros(eltype(first(u)), num_plotting_points, md.num_elements)
+  for e in Base.OneTo(md.num_elements)
+    # interpolate solution to plotting nodes element-by-element
+    StructArrays.foreachfield(mul_by_Vp!, u_plot_local, view(u, :, e))
+    for (i, u_i) in enumerate(u_plot_local)
+      # transform nodewise solution according to `solution_variables`
+      u_plot[i, e] = solution_variables(u_i, equations)[variable_id]
+    end
+  end
+
+  legend --> false
+  aspect_ratio --> 1
+  title --> pd.variable_names[variable_id]
+  xlims --> extrema(md.x)
+  ylims --> extrema(md.y)
+  seriestype --> :heatmap
+
+  return DGTriPseudocolor(plotting_triangulation(u_plot, rd.rstp, (x->rd.Vp * x).(md.xyz))...)
+end
+
+struct DGMultiPlotMesh{Dim, PD<:DGMultiPlotData{Dim}}
+  pd::PD
+end
+
+getmesh(pd::DGMultiPlotData) = DGMultiPlotMesh(pd)
+
+using StartUpDG: vandermonde, nodes
+
+# Visualize a 2D mesh associated with a DGMulti solver.
+# Note: This is an experimental feature and may be changed in future releases without notice.
+@recipe function f(pm::DGMultiPlotMesh{2})
+  @unpack pd = pm
+  @unpack rd, md = pd
+
+  x, y = plotting_wireframe(rd, md)
+
+  xlims --> (x[begin], x[end])
+  ylims --> (y[begin], y[end])
+  aspect_ratio --> :equal
+  legend -->  :none
+
+  # Set series properties
+  seriestype := :path
+  linecolor := :grey
+  linewidth := 1
```

Comment:
```suggestion
  seriestype --> :path
  linecolor --> :grey
  linewidth --> 1
```
This allows users to change these defaults when calling `plot(...)`. This is really helpful when trying to create plots for publications (where I often want to increase the `linewidth`).

Reply:
Thanks! I didn't know that.